python/dgl/transforms/functional.py

```diff
@@ -3672,7 +3672,7 @@ def lap_pe(g, k, padding=False, return_eigval=False):
     L = sparse.eye(g.num_nodes()) - N * A * N
 
     # select eigenvectors with smaller eigenvalues O(n + klogk)
-    EigVal, EigVec = np.linalg.eig(L.toarray())
+    EigVal, EigVec = scipy.sparse.linalg.eigs(L, k=pos_enc_dim+1, which='SR', tol=1e-2)
```

Comment:
Minor correction here: 
replace `k=pos_enc_dim+1` with `k=k+1` given the corresponding argument `k` passed in this function.
thanks @BarclayII!